_wiki/Install/Linux.md

```diff
@@ -65,6 +65,8 @@ systemctl --user daemon-reload
 systemctl --user enable opentabletdriver --now
 ```
 
+If the systemd user service does not start consistently, consider adding `otd-daemon` to your WM or DE's autostart.
```

Comment:
You can make an autostart `.desktop` file, assuming the DE/WM supports it
https://specifications.freedesktop.org/autostart-spec/autostart-spec-latest.html

Reply:
Thanks, opened a new issue regarding that: #43